### ISSUE TITLE
Bug - [DescriptionListTermHelpTextButton] - [a11y: Unable to open popover with keyboard]

### ISSUE THREAD

**Issue by ChrisLee5000** [patternfly/patternfly-react#10350]
**Describe the problem**
This is an issue with 
- [X ] Patternfly 5  
- [ ] Patternfly 6  
- [ ] other

Please give a clear and concise description of the problem. Which components are affected?

**How do you reproduce the problem?**
https://www.patternfly.org/components/description-list/react/term-help-text/

Use the keyboard to tab into the popover heading, and press enter to activate.

**Expected behavior**
Popover opens.

**Is this issue blocking you?**

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**
 - OS: Win 10
 - Browser Chrome
 - Version Latest
